controllers/seed-controller.js

```diff
@@ -58,4 +58,19 @@ exports.getArticleComments = (req, res, next) => {
     .catch((err) => {
         next(err)
     })
+}
+
+exports.postComment = (req, res, next) => {
+    const {article_id} = req.params
+    Promise.all([
+        insertComment(article_id, req.body), 
+        checkArticleExists(article_id),
+        checkUserExists(req.body)
```

Comment:
You shouldn't actually _**need**_ to check the existence of both the article and the user as PSQL should error out for us anyway. I can see that you have already done a 400 test for one of these occurrences